.github/filters.yml

```diff
@@ -58,6 +58,10 @@ blast_makeblastdb:
   - software/blast/makeblastdb/**
   - tests/software/blast/makeblastdb**
 
+blast_blastn:
+  - software/blast/blastn/**
+  - tests/software/blast/blastn**
```

Comment:
```suggestion
  - tests/software/blast/blastn/**
```

Reply:
Seems to work without it but for consistency can you add this for all other instances in this file too please @JoseEspinosa ?